hbase-server/src/main/java/org/apache/hadoop/hbase/client/ShortCircuitConnectionRegistry.java

```diff
@@ -68,6 +68,11 @@ public ShortCircuitConnectionRegistry(ConnectionRegistryEndpoint endpoint) {
     return future;
   }
 
+  @Override
+  public String getConnectionString() {
+    return "localhost";
```

Comment:
Maybe just call it short-circuit? I do not think localhost is accurate enough, it may implies that we are connecting to a regsitry through network but on the same machine? For short-circuit we know that we are using the ShortCircuitConnectionRegistry.

Reply:
This is a good point; `localhost` does have a specific meaning. Let me consider it, perhaps I can propose a better name. "short-circuit" isn't too bad.